service/README.rst

```diff
@@ -1,84 +1,92 @@
 
-GEOPM Systemd Service
-=====================
+GEOPM Service
+=============
 
+Features
+--------
 
-*
-  Fine grained access management system for hardware features: read
-  hardware "signals" and write hardware "controls".
+Linux Integration
+~~~~~~~~~~~~~~~~~
+  Linux Systemd Service with vendor agnostic DBus interface for
+  user-level access to hardware features on hetergenous systems
 
-*
-  System administrator configures access permissions based on Unix
-  group.
 
-*
-  Save / restore of hardware controls based on client process lifetime.
+Hardware Telemetry
+~~~~~~~~~~~~~~~~~~
+  Users read telemetry from hardware components on heterogeneous
+  systems
 
-*
-  A high performance interface enabling batch access to validated sets
-  of signals/controls.
 
-*
+Hardware Configuration
+~~~~~~~~~~~~~~~~~~~~~~
+  Users make ephemeral configurations to hardware component device
+  settings on heterogeneous systems
+
+
+Quality of Service
+~~~~~~~~~~~~~~~~~~
+  GEOPM Service reverts any changes made to hardware configurations
+  after the client's Linux process session ends
```

Comment:
Do these two describe the same feature? If so, I think the second one is more specific and the first can be removed. If not, maybe add an example.

Reply:
These two are clearly related.  If the word ephemeral were removed from the sentence above, they would maybe be more clearly different?  The first feature is for users who want to modify hardware settings, the second feature is for system administrators who would prefer that the previous feature to doesn't cause QoS problems for other users.

Reply:
Ah that makes sense. Yeah, maybe dropping the persistence note from the first one would help keep the focus on the configuration aspect of it.